pkg/controller/operands/kubevirt.go

```diff
@@ -24,15 +24,17 @@ const (
 )
 
 const (
-	kvHotplugVolumes = "HotplugVolumes"
-	cmFeatureGates   = "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot"
+	kvHotplugVolumes         = "HotplugVolumes"
+	kvWithHostPassthroughCPU = "WithHostPassthroughCPU"
```

Comment:
Can we add `kvWithHostModelCPU = "WithHostModelCPU"? We need to manage them both.